Tasks/ServiceFabricDeploy/deploy.ps1

```diff
@@ -31,6 +31,7 @@ try {
     $compressPackage = [System.Boolean]::Parse((Get-VstsInput -Name compressPackage))
     $skipUpgrade =  [System.Boolean]::Parse((Get-VstsInput -Name skipUpgradeSameTypeAndVersion))
     $skipValidation =  [System.Boolean]::Parse((Get-VstsInput -Name skipPackageValidation))
+    $unregisterUnusedVersions = [System.Boolean]::Parse((Get-VstsInput -Name unregisterUnusedVersions))
```

Comment:
Missed that but, still facing this issue.

![image](https://cloud.githubusercontent.com/assets/11148158/24574764/14eb58e8-16b5-11e7-9f4f-8cb00c7d4008.png)